.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,32 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Add label and request review
+        run: |
+          reviewers="@cryostatio/reviewers"
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
+
+          if [ -n "$reviewers" ]; then
+            gh pr request-review ${{ github.event.issue.number }} --reviewer $reviewers
```

Comment:
I don't believe there's a request-review command... I think it's `gh pr create --reviewer `

Reply:
I'm pretty sure that's going to trigger creation of a new PR.

Reply:
`gh pr edit --add-reviewer` should do it.

Reply:
[https://github.com/aali309/cryostat/actions](https://github.com/aali309/cryostat/actions)
yes, after testing it for a while with @mwangggg and adding her as a contributor on my repo it works now